sdk/cosmos/src/clients/database_client.rs

```diff
@@ -59,14 +79,90 @@ impl DatabaseClient {
         Ok(GetDatabaseResponse::try_from(response).await?)
     }
 
-    /// List collections in the database
-    pub fn list_collections(&self) -> requests::ListCollectionsBuilder<'_> {
-        requests::ListCollectionsBuilder::new(self)
+    /// Delete the database
+    pub async fn delete_database(
+        &self,
+        ctx: Context,
+        options: DeleteDatabaseOptions,
+    ) -> crate::Result<DeleteDatabaseResponse> {
+        let mut request = self
+            .cosmos_client()
+            .prepare_request_pipeline(&format!("dbs/{}", self.database_name()), http::Method::GET);
```

Comment:
This GETs the database, it doesn't delete it at all!

Will be fixed in #375 once we catch up with `main` and update that MR, if you don't get to it first.